ibis/expr/operations/relations.py

```diff
@@ -408,29 +408,40 @@ class SQLQueryResult(Relation):
 
 
 @public
-class SQLStringView(PhysicalTable):
-    """A view created from a SQL string."""
+class View(PhysicalTable):
+    """A view created from an expression."""
 
     child: Relation
-    query: str
 
     @attribute
     def schema(self):
-        # TODO(kszucs): avoid converting to expression
-        backend = self.child.to_expr()._find_backend()
-        return backend._get_schema_using_query(self.query)
+        return self.child.schema
 
 
 @public
-class View(PhysicalTable):
-    """A view created from an expression."""
+class SQLStringView(Relation):
+    """A view created from a SQL string."""
 
-    child: Relation
-    name: str
+    child: View
```

Comment:
Then this could be `NamedRelation`

Reply:
Can be done in a follow up if someone is motivated to do so.